src/client_hints/save_data.rs

```diff
@@ -0,0 +1,117 @@
+use crate::bail_status;
+use crate::headers::{HeaderName, HeaderValue, Headers, ToHeaderValues, SAVE_DATA};
+
+use std::fmt::Debug;
+use std::option;
+
+/// HTTP `SaveData` header
+///
+/// This header is considered "experimental" and may be subject to change as the
```

Comment:
I think we should put this behind e.g. `"unstable"`...

Reply:
Hmm, yeah fair -- or perhaps just not include them for now since they _are_ experimental. I'm so-so about it.